app/assets/stylesheets/frontend/views/_how-gov-works.scss

```
@@ -213,8 +213,20 @@
     border-width: 0 0 1px;
   }
 
-  @media (min-width: 640px) and (max-width: 900px) {
-    .prime-minister,
+  .prime-minister,
+  .cabinet {
+    .one-half {
+      &:first-child {
+        float: right;
+      }
+
+      &:nth-child(2) {
+        float: left;
+      }
```

I really like that you've used (good old-fashioned) floats here as it works all the way back to older versions of Internet Explorer. 

Since IE 8 doesn't support the `nth-child()` selector and this still works, I wonder if the `nth-child` selector is needed? Removing the `float:left` doesn't cause any change in layout using the latest versions of Firefox, Chrome, or Safari.
```suggestion
      &:first-child {
        float: right;
      }
```